src/grisp_connect_ws.erl

```diff
@@ -99,6 +105,10 @@ handle_info({gun_ws, Pid, Stream, {close, Code, Message}},
             #state{gun_pid = Pid, ws_stream = Stream} = S) ->
     ?LOG_WARNING(#{event => stream_closed, code => Code, reason => Message}),
     {noreply, S};
+handle_info({gun_error, Pid, _Stream, Reason}, #state{gun_pid = Pid} = S) ->
+    ?LOG_INFO(#{event => ws_closed, reason => Reason}),
```

Comment:
Can you replace `ws_closed` with `gun_error` for the event field? I think it is more appropriate.

Reply:
Also, I would use LOG_ERROR

Reply:
ok, for it being a LOG_ERROR, what I am confused with it why gun_error instead of ws_close ?? I see that everywhere that the connection is closed it use the event ws_close even though the trigger is gun_down, so my idea was that the even represent the outcome, not the trigger, and in case of a a gun_error the outcome. If I change it to gun_error, the logic would be to change the other to gun_down...

Reply:
I used ws_cloded because in the match head I match ws atom from gun.
In this case I'd like to underline that this is a `gun_error` and not just some gun reaction.

Reply:
But LOG_ERROR is enough to make it distinct, I'm fine with ws_closed and LOG_ERROR